wire-library/wire-tests/src/commonTest/shared/json/all_64_min_proto3.json

```diff
@@ -44,5 +44,20 @@
     "-9223372036854775808",
     "-9223372036854775808"
   ],
-  "oneofInt64": "-9223372036854775808"
+  "oneofInt64": "-9223372036854775808",
+  "mapInt64Int64": {
+    "-9223372036854775808": "-9223372036854775808"
+  },
+  "mapInt64Uint64": {
+    "-9223372036854775808": "9223372036854775808"
+  },
+  "mapInt64Sint64": {
+    "-9223372036854775808": "-9223372036854775808"
+  },
+  "mapInt64Fixed64": {
+    "-9223372036854775808": "9223372036854775808"
```

Comment:
I would have expected this to be 0? unless we’re testing that it is backwards-compatible

Reply:
This is the JSON generated by protoc

Reply:
Right, will update what we test in followup